include/ddc/chunk_span.hpp

```diff
@@ -45,6 +43,11 @@ template <class ElementType, class... DDims, class LayoutStridedPolicy, class Me
 class ChunkSpan<ElementType, DiscreteDomain<DDims...>, LayoutStridedPolicy, MemorySpace>
     : public ChunkCommon<ElementType, DiscreteDomain<DDims...>, LayoutStridedPolicy>
 {
+    static_assert(
+            std::is_same_v<LayoutStridedPolicy, Kokkos::layout_left>
+            || std::is_same_v<LayoutStridedPolicy, Kokkos::layout_right>
+            || std::is_same_v<LayoutStridedPolicy, Kokkos::layout_stride>);
```

Comment:
```suggestion
            || std::is_same_v<LayoutStridedPolicy, Kokkos::layout_stride>, "ChunkSpan only supports layout_left, layout_right or layout_stride");
```

Reply:
I agree, I will update it

Reply:
fixed in 49c89244b2b6db06209098c6cf4c5a9676f8b5ae